spec/lib/protobuf/varint_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'base64'
+require 'spec_helper'
+
+RSpec.describe Protobuf::Varint do
+  VALUES = {
```

Comment:
I picked pretty arbitrary values to test that these are working. If someone has another suggestion, happy to include something else.